src/jit/lower.cpp

```diff
@@ -3679,11 +3679,10 @@ GenTree* Lowering::LowerSignedDivOrMod(GenTreePtr node)
         return next;
     }
 
-    // We're committed to the conversion now. Go find the use.
+    // We're committed to the conversion now. Go find the use if any.
     LIR::Use use;
     if (!BlockRange().TryGetUse(node, &use))
     {
-        assert(!"signed DIV/MOD node is unused");
```

Comment:
Yeah, I haven't been able to get a small repro for this. It's pretty tricky to sneak an unused divide-by-constant into lowering.